src/mixins/common-mixin.ts

```diff
@@ -145,10 +145,8 @@ export default class CommonMixin extends Vue {
       case RouteNames.CHANGE: return 'Business Information'
       case RouteNames.CONVERSION: return 'Record Conversion'
       case RouteNames.CORRECTION: return 'Register Correction'
-      case RouteNames.RESTORATION: {
-        if (this.isLimitedExtendRestorationFiling) return 'Limited Restoration Extension'
-        if (this.isLimitedConversionRestorationFiling) return 'Conversion to Full Restoration'
-      }
+      case RouteNames.RESTORATION_EXTENSION: return 'Limited Restoration Extension'
+      case RouteNames.RESTORATION_CONVERSION: return 'Conversion to Full Restoration'
```

Comment:
I'm not sure we need this method.  As far as I can see, it's only used to set the title in the view.  I'd suggest we'd be better to add the text directly to the view.

Reply:
It's here as a common pattern (see methods above and also in other apps), but in the vein of having all filing-specific items in a single place (view), I'm OK with moving this (now or later).

Reply:
The pattern is having helper functions in a mixin to convert a type/route/etc to a string, for example.